components/overflow-group/overflow-group-mixin.js

```diff
@@ -382,6 +382,7 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 			}
 
 			this._chomp();
+			this.requestUpdate();
```

Comment:
It's possible to have situations where the `_chompIndex` doesn't change but we need to re-render (e.g., new filter is added but chomp index is calculated to stay the same we still need to re-render in order to add `data-is-chomped` to applicable elements)

Reply:
Minor: Are there still situations where `this._chomp` doesn't need an update?  Seems like every `_chomp` now requests an update or happens in `update`.  Just wondering if this should go in `_chomp`, but maybe not worth possible issues or regression with that

Reply:
That's a good point. I think actually with the `_handleResize` case we could get rid of it and that would probably be a good idea for performance. The slot change and mutation cases would be the ones where it would be more likely to be needed.